src/index.html

```diff
@@ -4,11 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+        src="moyo_1.png"
+        alt="MOYO"
+        class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="item"><a class="link is-active" href="#">APPLE</a></li>
+          <li class="item"><a class="link" href="#">SAMSUNG</a></li>
+          <li class="item"><a class="link" href="#">SMARTPHONES</a></li>
+          <li class="item">
+            <a
+            class="link"
+            href="#"
+            data-qa="hover">LAPTOPS & COMPUTERS</a>
+          </li>
+          <li class="item"><a class="link" href="#">GADGETS</a></li>
+          <li class="item"><a class="link" href="#">TABLETS</a></li>
+          <li class="item"><a class="link" href="#">PHOTO</a></li>
+          <li class="item" ><a class="link" href="#">VIDEO</a></li>
+        </ul>
+      </nav>
+
+    </header>
+
+
```

Comment:
please pay attention to formatting your code. look at [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

Reply:
not fixed!